src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  value: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+
+  reducers: {
+    addbook: { },
+    removebook: { },
+
+  },
```

Comment:
- **[OPTIONAL]** Your reducers should have actions to addbook and removebook, I mean, please 🙏🏽  complete the actions to successfully add and remove a book, you can see this [video tutorial](https://youtu.be/bbkBuqC1rU4?t=3446).